org-super-agenda.el

```diff
@@ -250,7 +261,7 @@ Prepended with `org-super-agenda-header-separator'."
                       ;; testing, it only takes effect in Agenda buffers when `local-map' is set, so
                       ;; we'll use both.
                       'local-map org-super-agenda-header-map)
-       (concat org-super-agenda-header-separator s))))
+       (concat (org-super-agenda--make-agenda-header-separator) s))))
```

Comment:
Since this is only done in one place, let's not make a new function.  Instead, please put the `cl-typecase` here directly.